<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliceDialog</class>
 <widget class="QDialog" name="SliceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slice</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Table preview</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Camera:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="camFrontBtn">
              <property name="text">
               <string>Front</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="camSideBtn">
              <property name="text">
               <string>Side</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="camTopBtn">
              <property name="text">
               <string>Top</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="stlViewWidget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="consoleGroup">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Console</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QPlainTextEdit" name="console"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="loaderLabel">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
color: #ffffff;
text-align: center;
 }

QProgressBar:horizontal {
border-radius: 10px;
background-color:qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(67, 67, 67, 255), stop:0.5 rgba(82, 82, 82, 255), stop:0.903955 rgba(74, 74, 74, 255), stop:1 rgba(140,140,140, 255));
padding: 3px;
}
QProgressBar::chunk:horizontal {
border: 1px solid #44445F;
border-radius: 7px;
background:  qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 #8888BF, stop:0.559322 #4E4E7F, stop:1 #9696BF);
}
</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Slicer options</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Config:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="confCombo"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Layer height:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="layerHeight">
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Infill:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="fillDensity">
           <property name="suffix">
            <string>%</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Object manipulation</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="addBtn">
           <property name="text">
            <string>Load File</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/imgs/icons/open.png</normaloff>:/imgs/icons/open.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Tools</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="3">
             <widget class="QToolButton" name="selectTool">
              <property name="toolTip">
               <string>Select</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/cursor.png</normaloff>:/imgs/icons/cursor.png</iconset>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>X:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="xSpinBox">
              <property name="suffix">
               <string> cm</string>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="minimum">
               <double>-100.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="0" column="2" rowspan="10">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QToolButton" name="boxTool">
              <property name="toolTip">
               <string>Box select</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/boxselect.png</normaloff>:/imgs/icons/boxselect.png</iconset>
              </property>
             </widget>
            </item>
            <item row="7" column="5">
             <widget class="QToolButton" name="repairTool">
              <property name="toolTip">
               <string>Repair</string>
              </property>
              <property name="text">
               <string>Repair</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/repair.png</normaloff>:/imgs/icons/repair.png</iconset>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="3">
             <widget class="QToolButton" name="mirrorTool">
              <property name="toolTip">
               <string>Mirror</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/mirror.png</normaloff>:/imgs/icons/mirror.png</iconset>
              </property>
             </widget>
            </item>
            <item row="6" column="5">
             <widget class="QToolButton" name="cloneTool">
              <property name="toolTip">
               <string>Clone</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/stamp.png</normaloff>:/imgs/icons/stamp.png</iconset>
              </property>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QToolButton" name="centerTool">
              <property name="toolTip">
               <string>Center</string>
              </property>
              <property name="statusTip">
               <string>Center</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/center.png</normaloff>:/imgs/icons/center.png</iconset>
              </property>
             </widget>
            </item>
            <item row="4" column="5">
             <widget class="QToolButton" name="toolButton">
              <property name="toolTip">
               <string>Scale</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/scale.png</normaloff>:/imgs/icons/scale.png</iconset>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QToolButton" name="rotateTool">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="toolTip">
               <string>Rotate</string>
              </property>
              <property name="text">
               <string>Rotate</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/rotate.png</normaloff>:/imgs/icons/rotate.png</iconset>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="5">
             <widget class="QToolButton" name="moveTool">
              <property name="toolTip">
               <string>Move</string>
              </property>
              <property name="text">
               <string>Move</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/drag.png</normaloff>:/imgs/icons/drag.png</iconset>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QToolButton" name="delTool">
              <property name="toolTip">
               <string>Delete</string>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/delete.png</normaloff>:/imgs/icons/delete.png</iconset>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QSpinBox" name="rotationsSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="suffix">
               <string>°</string>
              </property>
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Rotation:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Scale:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="scaleSpinBox">
              <property name="suffix">
               <string>%</string>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Y:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="ySpinBox">
              <property name="prefix">
               <string/>
              </property>
              <property name="suffix">
               <string> cm</string>
              </property>
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="minimum">
               <double>-100.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item row="8" column="3" colspan="3">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="9" column="3">
             <widget class="QToolButton" name="sliceTool">
              <property name="toolTip">
               <string>Slice</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/slice.png</normaloff>:/imgs/icons/slice.png</iconset>
              </property>
             </widget>
            </item>
            <item row="9" column="5">
             <widget class="QToolButton" name="materialTool">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/imgs/icons/fill.png</normaloff>:/imgs/icons/fill.png</iconset>
              </property>
              <property name="popupMode">
               <enum>QToolButton::DelayedPopup</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Export</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="sliceBtn">
           <property name="text">
            <string>GCode</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/imgs/icons/layers.png</normaloff>:/imgs/icons/layers.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportStlBtn">
           <property name="text">
            <string>STL</string>
           </property>
           <property name="icon">
            <iconset resource="resource.qrc">
             <normaloff>:/imgs/icons/cube.png</normaloff>:/imgs/icons/cube.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
